src/widget/reorderableList.ts

```diff
@@ -232,18 +232,20 @@ export class DropPlaceholder extends St.Widget {
             'drag-over': {},
         },
     };
-    private _delegate: this;
```

Comment:
Private is bad practice?

Reply:
It should not be private. It's used by the gnome shell. Typescript complains that the field is never read from otherwise.